go.mod

```diff
@@ -78,3 +78,5 @@ require (
 )
 
 replace github.com/nats-io/go-nats v1.4.0 => github.com/mysteriumnetwork/nats.go v1.4.1-0.20200303115848-b4a5324c56ed
+
+replace github.com/mysteriumnetwork/payments => ../payments
```

Comment:
dont think this needs to get merged

Reply:
yep. That is why this PR is draft. It will be changed once the payments changes merged.